contrib/Dockerfile.manylinux

```
@@ -362,6 +362,23 @@ ARG UCX_SONAME_SUFFIX=""
+# Opt-in: place the pre-staged DDN Infinia libs (from infinia-libs/ in the build
+# context, populated by build-container.sh --build-infinia) into /opt/ddn/red so
+# meson auto-detects red_client and builds libplugin_INFINIA.so into the wheel.
+# libred_client/libred_async are excluded from the final wheel by auditwheel
+# (build-wheel.sh AUDITWHEEL_EXCLUDES) — only libplugin_INFINIA.so is bundled;
+# the DDN libs are loaded at runtime from the customer's installation.
+# No external registry pull here: libs arrive via COPY . /workspace/nixl above.
+# libuuid-devel provides uuid/uuid.h, included by the DDN red_client headers;
+# it is the only extra build-time system dep the plugin needs (the DDN libs'
+# own runtime deps are provided by the customer's install and excluded by auditwheel).
+ARG BUILD_INFINIA=false
+RUN if [ "$BUILD_INFINIA" = "true" ]; then \
```

Hadolint (2.14.0)

[warning] Specify version with `dnf install -y <package>-<version>`.

(DL3041)